File: Java Script - Introduction to Programming/4.1. Complex Conditions/09. Day of Week.js
function dayOfWeek([...arg]) {
  let n = Number(arg[0]);
  let day = ' ';
  switch (n) {
    case 1: day = 'Monday'; break;
    case 2: day = 'Tuesday'; break;
    case 3: day = 'Wednesday'; break;
    case 4: day = 'Thursday'; break;
    case 5: day = 'Friday'; break;
    case 6: day = 'Saturday'; break;
    case 7: day = 'Sunday'; break;
    default:
      return 'Error'
  }

return day  
}

// console.log(dayOfWeek([4]))

File: Pyrhon - Introduction to Programming/4.1. Complex Conditions/09. Day of Week.py
def get_day(day):
  switcher = {
      1: 'Monday',
      2: 'Tuesday',
      3: 'Wednesday',
      4: 'Thursday',
      5: 'Friday',
      6: 'Saturday',
      7: 'Sunday',
  }

  return switcher.get(day , 'Error')


print(get_day(int(input())))


'''
1
#Monday
'''
